test/e2e/tests/bridge/bridge-test-utils.ts

```diff
@@ -59,44 +57,12 @@ export class BridgePage {
     );
   };
 
-  navigateToAssetPage = async (
-    contractRegistry: GanacheContractAddressRegistry,
-    symbol: string,
-    shouldImportToken = true,
```

Comment:
we don't need to import the token manually, we can use fixtures for that. This way, this method sticks to what it's named: just `navigateToAssetPage`